variables.tf

```diff
@@ -387,3 +387,635 @@ variable "aft_metrics_reporting" {
     error_message = "Valid values for var: aft_metrics_reporting are (true, false)."
   }
 }
+
+#########################################
+# local_file replacement variables
+#########################################
+
+variable "aft_version" {
+  type    = string
+  default = "1.11.1"
+}
+
+variable "python_version" {
+  type    = string
+  default = "3.11"
+}
+
+variable "ct_aft_account_provisioning_customizations" {
```

Comment:
This looks a bit ugly.
Have we tried > https://developer.hashicorp.com/terraform/language/functions/file ?
Do we know why upstream chose to use local_file?